test/metabase/channel/http_test.clj

```diff
@@ -328,10 +328,10 @@
          :model/Channel      {chn-id :id}  {:type    :channel/http
                                             :details {:url         (str url (:path receive-route))
                                                       :auth-method "none"}}
-         :model/PulseChannel _             {:pulse_id     pulse-id
+         :model/PulseChannel {pc-id :id}   {:pulse_id     pulse-id
                                             :channel_type "http"
                                             :channel_id   chn-id}]
-        (pulse/send-pulse! pulse)
+        (#'task.send-pulses/send-pulse!* pulse-id [pc-id])
```

Comment:
it's more _e2e_ now.

It's not truly e2e if we don't use the actual scheduler, but I think that's excessive, considering we have tests that check tasks are correctly scheduled.